### PR TITLE
setup readthedocs

### LINKED ISSUE #425: Explorer needs a readthedoc site
There are many app configurations that needs to be explained in full detail, the current readme isn't sufficient

### PR DESCRIPTION
**Pull request by pindge**
## Scope 
- setup readthedoc for explorer

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the ODC Steering Council Chair,
+currently [Alex Leith](alex.leith@ga.gov.au) or alternatively, a member of
```
Comment (pindge) [inline code-of-conduct.md]: @SpacemanPaul is this correct?

File: docs/config.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+.. py:data:: CUBEDASH_PRODUCT_GROUP_BY_REGEX
+
+    Tuple containing regex for product name matching and group name
+
+    Default: ``None``
+    Example: ``((r'^usgs_','USGS products'), (r'_albers$','C2 Albers products'), (r'level1','Level 1 products'), )``
+
+.. py:data:: CUBEDASH_PRODUCT_GROUP_BY_FIELD
+
+    ??
```
Comment (pindge) [inline docs/config.rst:25]: @SpacemanPaul shoud I put # TODO instead? I am unclear about what those do, need Jeremy to write it up

File: docs/config.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    Enable Cross Origin Resource Sharing (CORS) for ``stac`` and ``api``.
+
+    Default: ``True``
+
+.. py:data:: CUBEDASH_PRODUCT_GROUP_BY_REGEX
+
+    Tuple containing regex for product name matching and group name
+
+    Default: ``None``
+    Example: ``((r'^usgs_','USGS products'), (r'_albers$','C2 Albers products'), (r'level1','Level 1 products'), )``
```
Comment (pindge) [inline docs/config.rst:21]: @SpacemanPaul , the default is `none`, but i'd like to give example, does this explanation look ok?

File: code-of-conduct.md (hunk 1 above)
Comment (robbibt) [reply code-of-conduct.md]: The current ODC Steering Council chair is Caitlin Adams (cadams@frontiersi.com.au); not sure about the secretariat...

Comment (pindge) [reply code-of-conduct.md]: thanks @robbibt

**Review by SpacemanPaul — APPROVED** [review]
Good idea.

File: docs/config.rst (hunk 3 above)
Comment (SpacemanPaul) [reply docs/config.rst:21]: Makes sense to me.

File: docs/config.rst (hunk 2 above)
Comment (SpacemanPaul) [reply docs/config.rst:25]: I think I'd prefer "TODO".

**Pull request merged**